src/views/CTLViewsHoC.jsx

```diff
@@ -82,6 +80,9 @@ const CTLViewsHoC = (WrappedView) => {
 					navigate(
 						`${CodeOrPurlPath({ codeOrPurl: prettyUrlName })}${queryString}`
 					);
+				} else {
```

Comment:
I don't believe you need this else block. Even if you need line `84`(which I believe you can do without), you don't need line `85`. Remove this else block and test this.